src/main/java/ui/workout/workoutsession/WorkoutSessionUi.java

```diff
@@ -48,11 +48,11 @@ public static void printHelp() {
     }
 
     public static void emptyInputError() {
-        System.out.println("Please enter something.");
+        printer.showToUser("Please enter something.");
     }
 
     public static void searchInputError() {
-        printer.showToUser("Wrong format, please enter in the format: search [name of move]");
+        printer.showToUser("Wrong format, please enter in the format:\n\t search [name of move]");
```

Comment:
And also here to "search [NAME_OF_MOVE]"

Reply:
will do